Automattic/wp-calypso
# Make Stats Commercial "Annual" plan the default billing term.

This PR makes the Stats Commercial **yearly/annual** billing term the default plan when selected to purchase (as opposed to the monthly billing term product).



## Proposed Changes

* Makes the necessary code modifications to make the Stats Commercial **yearly** plan as default.

## Testing Instructions



* If code-D118168 is not merged yet, you will first need to apply patch code-D118168 to your sandbox, and sandbox `public-api.wordpress.com`, and `define( 'USE_STORE_SANDBOX' );` in `0-sandbox.php` file.
* Checkout this PR and run `yarn start`.
* Go to: http://calypso.localhost:3000/stats/purchase/:SITE  (where `:SITE` is the site slug of a self-hosted Jetpack site connected to your account)
* Select "Commercial site" and verify the Stats Commercial "yearly" billing product is shown at $8.33/mo USD. (See screenshot).
![Markup 2023-08-08 at 16 44 29](https://github.com/Automattic/wp-calypso/assets/11078128/41dd89f7-9247-4b77-bf6e-79054e2f4c67)

* Select the "Get Jetpack Stats" button and verify you are taken to checkout with the "yearly" Stats Commercial product in the cart.
* Go back to the http://calypso.localhost:3000/stats/purchase/:SITE page and select "Personal site" this time. Verify the **Personal site** slider works the same as it always has prior to this PR.

**Bonus:**
* Test the Stats Commercial annual product in some other currencies (Change your currency in Store Admin) and verify the price display is accurate and looks fine.

Stats Commercial "Yearly/annual" prices are as follows:

| currency | per_unit_fee |
| -------- | ------------ |
| USD      | 100.00       |
| EUR      | 120.00       |
| AUD      | 156.00       |
| GBP      | 96.00        |
| BRL      | 408.00       |
| CAD      | 144.00       |
| CHF      | 108.00       |
| CZK      | 2400.00      |
| DKK      | 804.00       |
| HKD      | 804.00       |
| HUF      | 57588.00     |
| IDR      | 1919940.00   |
| ILS      | 384.00       |
| INR      | 5532.00      |
| JPY      | 12504.00     |
| MXN      | 1752.00      |
| NOK      | 1008.00      |
| NZD      | 180.00       |
| PHP      | 3852.00      |
| PLN      | 396.00       |
| RUB      | 7644.00      |
| SEK      | 1104.00      |
| SGD      | 156.00       |
| THB      | 2808.00      |
| TWD      | 3204.00      |
| TRY      | 696.00       |

.

## Pre-merge Checklist
